src/Nav.jsx

```diff
@@ -0,0 +1,34 @@
+import {useCart} from "./hooks/useCart";
```

Comment:
Why this file is in src folder and not in components? If there is no specific reason then you should move it to components folder